packages/gafl-webapp-service/src/pages/date-of-birth/route.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+import pageRoute from '../../page-route.js'
+import Joi from '@hapi/joi'
+
+export default pageRoute(
+  'date-of-birth',
+  '/date-of-birth',
```

Comment:
Can we use /buy/date-of birth - need to keep URL's as consistent with the existing service as possible.